### ISSUE TITLE
Add properties for spatial size

### ISSUE THREAD

**Issue by dr-shorthair** [opengeospatial/geosemantics-dwg#85]
Length, area, volume - e.g. see from GEOX

- [hasLength](https://raw.githack.com/CSIRO-enviro-informatics/geosparql-ext-ont/master/geox.html#haslength)
- [hasArea](https://raw.githack.com/CSIRO-enviro-informatics/geosparql-ext-ont/master/geox.html#hasarea)
- [hasVolume](https://raw.githack.com/CSIRO-enviro-informatics/geosparql-ext-ont/master/geox.html#hasvolume) 

Detail from #73

**Comment by mathib** [comment]
From the documentation, it seems that these properties are used on a feature of interest described by geometry? In the case of our approach for such (and other properties), we made it possible to relate such a property (if objectified) to the geometry it is derived/calculated from. Graph figure from the https://w3id.org/omg# documentation: https://www.projekt-scope.de/ontologies/omg/implicitProperty.png

This is again useful if you allow a certain feature of interest to have multiple geometry descriptions. If you add measures for versioning of the geometry, it even becomes possible to quickly check for such derived properties that might need an update.

**Comment by dr-shorthair** [comment]
Currently they are scoped to `geo:SpatialObject`, but I'm open to suggestions.

**Comment by mathib** [comment]
> Currently they are scoped to `geo:SpatialObject`, but I'm open to suggestions.

Right, and that's a super class of `geo:Feature` and `geo:Geometry`, thus it can be connected directly to something with a geometry description and the geometry description itself. If the property is connected to the geometry description, the property is probably also valid for the `geo:Feature`? I think this can be inferred using a property chain axiom.

If the property itself is modeled using n-ary relations, it is possible to link the property node to the source geometry:

```
# approach 1: property on the geometry description and inferred on the described object
inst:wall1 a bot:Element ;
    omg:hasGeometry inst:geometry-wall1 ;
    props:hasVolume inst:volume-wall1 . # inferred using prop chain or rule
inst:volume-wall1 a props:Property ;
    schema:value "10" ;
    omg:isImplicitlyDerivedFrom inst:geometry-wall1 .
inst:geometry-wall1 a omg:Geometry ;
    props:hasVolume inst:volume-wall1 ;
    fog:asObj_v3.0 "v 45.867954254150391 13.540400505065918 18.582895278930664\nv 45.867954254150391 ..." .
```

```
# approach 2: property directly on the described object instead of its geometry description
inst:wall1 a bot:Element ;
    omg:hasGeometry inst:geometry-wall1 ;
    props:hasVolume inst:volume-wall1 .
inst:volume-wall1 a props:Property ;
    schema:value "10" ;
    omg:isImplicitlyDerivedFrom inst:geometry-wall1 .
inst:geometry-wall1 a omg:Geometry ;
    fog:asObj_v3.0 "v 45.867954254150391 13.540400505065918 18.582895278930664\nv 45.867954254150391 ..." .
```

with:
* `inst:wall1` ~ instance of `geo:Feature`
* `inst:volume-wall1` ~ instance of `geox:SpatialMeasure`
* `inst:geometry-wall1` ~ instance of `geo:Geometry`

(Also see https://github.com/opengeospatial/geosemantics-dwg/issues/52 for the original input concerning properties related to geometry descriptions.)
